helm/dendrite/templates/deployment.yaml

```
@@ -0,0 +1,97 @@
+{{ template "validate.config" . }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: {{ $.Release.Namespace }}
+  name: {{ include "dendrite.fullname" . }}
+  labels:
+    app: {{ $.Chart.Name }}
```

put on every resource the labels by your helper:
```suggestion
   {{- include "dendrite.labels" . | nindent 4 }}
```